Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM rust:1.80.0-bullseye AS builder
+FROM rust:1.84.0-bullseye AS builder
```

Comment:
I think we can also bump the version in `lcli` too?

https://github.com/sigp/lighthouse/blob/96b00ef66c82eb7c67e76bf6142922e43f71f8b5/lcli/Dockerfile#L4

Reply:
Good catch! I thought of this and then forgot to do it 🤦‍♂️